app/helpers/application_helper.rb

```diff
@@ -6,4 +6,10 @@ module ApplicationHelper
   def new_session_path(*_)
     new_user_session_path
   end
+
+  def date_display(args)
+    Time.parse(args[:document][args[:field]])
+      .in_time_zone(Rails.configuration.time_zone)
```

Comment:
This works but is verbose and thankfully Rails has a method for this: `Time.zone.parse`

```ruby 
Time.parse(args[:document][args[:field]])
  .in_time_zone(Rails.configuration.time_zone)

### vs 

Time.zone.parse(args[:document][args[:field]])
```

Reply:
Another good one to know is `Time.zone.local` which you can use to construct new date-times in the local tz